### PR TITLE
ENG-810 Ad hoc card export

### PR DESCRIPTION
**Pull request by egh**
JIRA issue: https://jira.plos.org/jira/browse/ENG-810

#### What this PR does:

- Add basic ad-hoc card export
- Exports card attachments
- Includes the phase name in the task name for export to make which card it is clearer

#### Notes

I upgraded the ruby in `.ruby-version` to 2.4.10 because I could no longer install 2.3.x on my ubuntu 20.04 laptop.

---

#### Code Review Tasks:

**Author tasks** (delete tasks that don't apply to your PR, this list should be finished before code review):

**Reviewer tasks** (these should be checked or somehow noted before passing on to PO):
- [ ] I read through the JIRA ticket's AC before doing the rest of the review
- [ ] I ran the code (in the review environment or locally). I agree the running code fulfills the Acceptance Criteria as stated on the JIRA ticket
- [ ] I read the code; it looks good
- [ ] I have found the tests to be sufficient for both positive and negative test cases

### CODE REVIEW

File: app/views/export/_ad_hoc_element.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <% elsif ad_hoc_element["type"] == "attachments" %>
+    <%# attachments are exported elsewhere %>
+  <% elsif ad_hoc_element["type"] == "checkbox" %>
+    <% checked_str = ad_hoc_element["answer"] ? "checked" : "" %>
+    <p><input disabled type="checkbox" <%= checked_str %>> <%= ad_hoc_element["value"].try(:html_safe) %></p>
+  <% else %>
+    <% raise Exception.new("strange ad-hoc content: #{ad_hoc_element}") %>
+  <% end %>
+<% else %>
+  <% raise Exception.new("strange ad-hoc card") %>
```
Comment (sgatchell) [inline app/views/export/_ad_hoc_element.html.erb:22]: is there other context that would identify the oddball card?

**Comment by sgatchell** [comment]
looks good to me! Just had a question about strange cards but not a showstopper

**Review by sgatchell — APPROVED** [review]
:shipit:

File: app/views/export/_ad_hoc_element.html.erb (hunk 1 above)
Comment (egh) [reply app/views/export/_ad_hoc_element.html.erb:22]: It's really just to throw an error if we are looking at content that I didn't account for. I've exported a fair number of papers and they all seem fine, so it seems unlikely we'll hit these errors.

**Pull request merged**